src/backend/distributed/worker/worker_merge_protocol.c

```diff
@@ -17,7 +17,17 @@
 #include "funcapi.h"
 #include "miscadmin.h"
 
+#if PG_VERSION_NUM >= 120000
+#include "access/genam.h"
+#endif
 #include "access/htup_details.h"
+#if PG_VERSION_NUM >= 120000
+#include "access/table.h"
+#define heap_beginscan_catalog table_beginscan_catalog
```

Comment:
We should be consistent about these changes. For some other function in other files, we inline such changes in the code and here we're doing it here. We should try to do all the same, and I actually prefer the following, it makes it easy to navigate on eclipse.

```C
#if PG_VERSION_NUM >= 120000
	table_endscan(scan);
#else
	heap_endscan(scan);
#endif
```